index.bs

```diff
@@ -118,7 +125,7 @@ The following code queries the available local fonts, and logs details about eac
 // Asynchronous Query and Iteration
 (async () => { // Async block
   // May prompt the user:
-  const status = await navigator.permissions.request({ name: "local-fonts" });
+  const status = await navigator.permissions.request({ name: "font-access" });
```

Comment:
do we also want to change these from `request()` to `query()` ?

Reply:
Done. Changed the examples in the spec to align with the explainer.